.travis.yml

```diff
@@ -11,14 +11,26 @@ sudo: false
 
 cache:
   directories:
-    - $HOME/.composer/cache/files
+    - $HOME/.composer/cache
 
 services: mongodb
 
+matrix:
+   include:
+     - php: 5.6
+       env: SYMFONY_VERSION="2.3.*"
+     - php: 5.6
+       env: SYMFONY_VERSION="2.8.*"
+     - php: 5.6
+       env: SYMFONY_VERSION="3.0.*"
+
+before_install: if [[ "$SYMFONY_VERSION" != "" ]]; then composer require --no-update symfony/symfony:${SYMFONY_VERSION}; fi
+
+
 before_script:
   - tests/travis.sh
 
 script:
-  - phpunit
+  - phpunit --verbose
```

Comment:
I've added this when debugging the skipped tests, but slipped into the pr! Will remove it!